app/scripts/metamask-controller.js

```diff
@@ -545,10 +545,14 @@ export default class MetamaskController extends EventEmitter {
         this.networkController.getProviderAndBlockTracker().provider,
       // NOTE: This option is inaccurately named; it should be called
```

Comment:
Nit: I think we can drop this comment now. The fact that we are listening to `networkDidChange` and the fact that the callback takes the NetworkController state are unrelated.

Reply:
I agree those are unrelated, but I thought the comment was suggesting that the name should match the event. Right now it wrongly suggests that the handler should be called on each state change, when really it's called on each network switch.